scripts/test/compareBenchmarkScriptTest.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
```

Comment:
Is this test executed in CI? I can't see a change to the Jenkinsfile.

Edit: I noticed that it is called from other tests and doesn't have its own main. For me this means that it is not a script after all. This should be doced.

Reply:
It's included and called from `hyriseBenchmark[FileBased,TPCH,TPCC]_test.py`.

Reply:
See the edit.

Reply:
```
# This helper test class takes two JSON result files, runs the compare_benchmarks.py script, and checks that the output
# is as expected. It is called from benchmark tests such as hyriseBenchmarkTPCC_test.py.
```

Ok?

Reply:
Yepp